contracts/partial/yToken/lendingMethods.ligo

```diff
@@ -452,20 +432,13 @@ function liquidate(
             s.tokens
           );
 
-          if outstandingBorrowInCU <= liquidateCollateral
-          then failwith("yToken/liquidation-not-achieved");
-          else skip;
-          if borrowerAccount.borrow = 0n
-          then failwith("yToken/debt-is-zero");
-          else skip;
+          require(liquidateCollateral < outstandingBorrowInCU, "yToken/liquidation-not-achieved");
 
```

Comment:
Maybe it's duplicate, this check is lost.
```suggestion
require(borrowerAccount.borrow > 0n, "yToken/debt-is-zero");
```

Reply:
I removed it intentionally.
There's a check for underflow down the code.